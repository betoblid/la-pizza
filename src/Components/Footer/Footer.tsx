

export const Footer = () => {

    return (
        <footer className="bg-black py-2 text-center text-white">
            <p className="font-bold text-sm">
                Desenvolvido por
                <a href="https://www.linkedin.com/in/herbertlsouza"
                    target="_blank"
                    rel="noreferrer"
                    className="visited:text-red-300">
                    'Herbert Souza"</a></p>
        </footer>
    )
}