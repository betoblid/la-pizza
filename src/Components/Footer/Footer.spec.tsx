import { render, screen } from "@testing-library/react"
import { Footer } from "./Footer"

describe("<Footer />", () => {

    it("verificar se tem um paragrafo", () => {
        render(<Footer />)

        expect(screen.getByText(/Desenvolvido/i)).toBeInTheDocument()
    })
    it("verificar se tem um link", () => {
        render(<Footer />)
        expect(screen.getByRole("link", {name: /herbert/i})).toBeInTheDocument()
    })
    it("cirar um snapshot", () => {
        const { container } =  render(<Footer />)

        expect(container).toMatchSnapshot()
    })
})