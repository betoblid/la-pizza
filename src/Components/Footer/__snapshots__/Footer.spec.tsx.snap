// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> cirar um snapshot 1`] = `
<div>
  <footer
    class="bg-black py-2 text-center text-white"
  >
    <p
      class="font-bold text-sm"
    >
      Desenvolvido por
      <a
        class="visited:text-red-300"
        href="https://www.linkedin.com/in/herbertlsouza"
        rel="noreferrer"
        target="_blank"
      >
        'Herbert Souza"
      </a>
    </p>
  </footer>
</div>
`;
